### PR TITLE
newsletter 07-26-2019

### PR DESCRIPTION
**Pull request by ryan-shea**

### CODE REVIEW

**Comment by thelostone-mc** [comment]
^ @danlipert / @ryan-shea resolved conflicts -> could you guys check this out once more

File: app/retail/emails.py [hunk 1]
```diff
@@ -948,19 +948,19 @@ def render_new_bounty_roundup(to_email):
 Hey Gitcoiners,
 </p>
 <p>
-The Grow Ethereum Hackathon is right around the corner -- only 11 days remain until the hacking begins❗️ We are expecting great sponsors, including confirmation from the Ethereum Foundation itself (!) and UNICEF. Build projects together with top Ethereum companies and enterprises, win crypto, and unleash a new era of decentralized global infrastructure. The link to register is <a href="https://hackathons.gitcoin.co/grow-ethereum/" target="_blank">here.</a> 🌱
-</p>
+We are now only days away from the beginning of the Grow Ethereum hackathon, one of the largest hackathons we've planned to date. As we announced last week, we have huge names behind the event - the Ethereum Foundation and MetaCartel to name a couple - with dozens of prizes on the horizon. See the full list of sponsors (including bZx, AdEx, and Arweave) and a checklist to prepare for the July 29th start date on this <a href="https://gitcoin.co/blog/growethereum/" target="_blank">blog post.</a>
+/p>
```
Comment (danlipert) [inline app/retail/emails.py]: Hey @ryan-shea - lets try to lessen the number of typos in these - passing it through a simple linter or something might help. 🤔

File: app/retail/emails.py [hunk 2]
```diff
@@ -972,45 +972,45 @@ def render_new_bounty_roundup(to_email):
...
+        'primer': 'Allow Ability to Resurface Bounties to the Top When Expiration Date is Extended',
     }, {
-        'url': 'https://github.com/RibbonBlockchain/IncentivesMVP/issues/8',
-        'primer': 'Enable camera on front end for capture passport on patient and practitioner onboarding',
+        'url': 'https://github.com/SpeckleOS/speckle-browser-extension/issues/89',
+        'primer': 'Global Settings (Change Colour Theme and Log Out)',
     }, {
-        'url': 'https://github.com/centrifuge/precise-proofs/issues/88',
-        'primer': 'Optimize the fixed height tree implementation',
+        'url': 'https://gitcoin.co/issue/ironcoinext/IronCoin/8/3253',
```
Comment (danlipert) [inline app/retail/emails.py]: You have to use github.com links for this, not gitcoin.co links @vs77bb @ryan-shea

File: app/retail/emails.py [hunk 3]
```diff
@@ -972,45 +972,45 @@ def render_new_bounty_roundup(to_email):
...
         'body': [
-            'We just announced Tour de SOL, or our incentivized testnet event. Are you a validator? Earn token and race against the best in the world. Deadline is July 21st.'
+            'Bringing access, choice, and liquidity to unlisted infrastructure assets using the power of Ethereum'
         ]
     }
 
     bounties_spec = [{
-        'url': 'https://github.com/RibbonBlockchain/IncentivesMVP/issues/19',
-        'primer': 'Create Initial Pilot phase for Ribbon blockchain',
+        'url': 'https://github.com/gitcoinco/web/issues/4744',
```
Comment (danlipert) [inline app/retail/emails.py]: Why did y'all pick this one? It looks like a placeholder bounty for internal work or something like that @rafalkowalski @ryan-shea @vs77bb

**Comment by danlipert** [comment]
@thelostone-mc you don't need to rebase these, I just cherry-pick them. Also @ryan-shea please use a new branch for each newsletter

**Pull request closed without merging**